test/unit/TicketBroker.js

```diff
@@ -1737,19 +1524,28 @@ contract("TicketBroker", accounts => {
     })
 
     describe("claimableReserve", () => {
-        it("Returns 0 when the reserveHolder does not have a reserve", async () => {
+        it("returns 0 when the reserveHolder does not have a reserve", async () => {
             const numRecipients = 10
             await fixture.bondingManager.setMockUint256(functionSig("getTranscoderPoolSize()"), numRecipients)
             assert.equal((await broker.claimableReserve(constants.NULL_ADDRESS, constants.NULL_ADDRESS)).toString(10), "0")
         })
 
-        it("Returns 0 when the active transcoder pool size is 0 if reserve is not frozen", async () => {
+        it("returns 0 if claimant is not active in the current round", async () => {
             const reserve = 1000
             await broker.fundReserve({from: sender, value: reserve})
             assert.equal((await broker.claimableReserve(sender, constants.NULL_ADDRESS)).toString(10), "0")
         })
 
-        it("Returns claimable reserve for a claimaint if reserve is not frozen", async () => {
+        it("returns 0 when the active transcoder pool size is 0", async () => {
+            await fixture.bondingManager.setMockBool(functionSig("isRegisteredTranscoder()"), true)
```

Comment:
for the test-coverage , should this perhaps be 
`"isRegisteredTranscoder(address)"` to match the real function signature?

Reply:
Aha! Great catch. a43a2b4 should do the trick

Reply:
Yep it should, I ran coveralls locally already. 

Let's rebase ? 😄